pkg/mutation/schema/schema.go

```diff
@@ -0,0 +1,19 @@
+package schema
+
+import "github.com/open-policy-agent/gatekeeper/pkg/mutation/mutator"
+
+// DB caches all the implied schemas.
+// Will return an error when adding a mutator conflicting with the existing ones.
+type DB struct{}
```

Comment:
Do we want to call it `DB` or `Database`? Is DB common enough in usage that the chance of confusion is low?

Reply:
I feel DB (upper case) is enough, I can change the description to make it clearer.

Reply:
+1